pkcs11.c

```diff
@@ -663,7 +663,7 @@ QueryPkcs11DialogProc(HWND hwndDlg, UINT msg, WPARAM wParam, LPARAM lParam)
             {
                 DeleteObject(hf);
             }
-            RemoveProp(hwndDlg, cfgProp);
+            RemoveProp(hwndDlg, L"header_font");
```

Comment:
For `cfgProp` we use `const TCHAR *cfgProp = _T("conn");`, shouldn't we do the same for `header_font`?

Reply:
We can but its only used once so far...

Reply:
```
SetProp(parent, L"header_font", (HANDLE)hfb);	C:\Users\lev\Projects\openvpn-build\src\openvpn-gui\pkcs11.c	419	31
HFONT hf = (HFONT) GetProp(hwndDlg, L"header_font");	C:\Users\lev\Projects\openvpn-build\src\openvpn-gui\pkcs11.c	660	51
RemoveProp(hwndDlg, L"header_font");	C:\Users\lev\Projects\openvpn-build\src\openvpn-gui\pkcs11.c	665	35
```

Reply:
I know, "once" was used in plural :) (or all in one dialog)...  Well, I'm convinced: will define a variable.